build.gradle

```diff
@@ -1,24 +1,21 @@
-buildscript {
-    dependencies {
-        classpath 'com.diffplug.spotless:spotless-plugin-gradle:3.4.0'
-    }
-}
-
-// Used to promote a staging release build
-// https://github.com/Codearte/gradle-nexus-staging-plugin
-// gradle closeAndReleaseRepository
-plugins
-{
-    id "io.codearte.nexus-staging" version "0.8.0"
+plugins {
+	id 'java'
+	id 'maven'
+	id 'idea'
+	id 'signing'
+	id 'checkstyle'
+	id 'jacoco'
+	id "com.diffplug.gradle.spotless" version "3.4.0"
+	id 'org.sonarqube' version '2.6'
+	id 'com.google.protobuf' version '0.8.1'
```

Comment:
Why are we introducing protobuf plugin here?

Reply:
Good catch, this is a copy paste error. Will remove it in a later PR as this is harmless.